span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -152,7 +152,7 @@ public RawSpan convert(Span jaegerSpan) throws Exception {
             tenantId.get(),
             tenant ->
                 PlatformMetricsRegistry.registerTimer(
-                    SPAN_NORMALIZATION_TIME_METRIC, Map.of("tenantid", tenant)))
+                    SPAN_NORMALIZATION_TIME_METRIC, Map.of("tenantId", tenant)))
```

Comment:
note: @samarth-gupta-traceable we may have to make changes in our dashboard.

Reply:
got it .